SentryCapacitor.podspec

```diff
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
   s.author = package['author']
   s.source = { :git => package['repository']['url'], :tag => s.version.to_s }
   s.source_files = 'ios/Plugin/**/*.{swift,h,m,c,cc,mm,cpp}'
-  s.ios.deployment_target  = '12.0'
+  s.ios.deployment_target  = '13.0'
```

Comment:
My main concern here is  users using the SDK targeting iOS 12.0, on the best scenario we could target 12.0 for older Ionic versions and 13 for the latest version. What do you think about it?

Reply:
sounds nice 😊 

I wonder how to know the capacitor version

Reply:
How about leaving the `deployment_target` on 12.0 and relying on `@capacitor/core` to check if it's requirement is met?
According to [migration docs](https://capacitorjs.com/docs/updating/4-0#raise-ios-deployment-target) it should be built in `ios/App/Podfile`:

```
post_install do |installer|
  assertDeploymentTarget(installer)
end
```